tests/ui_tests/app/page/host_list/locators.py

```diff
@@ -0,0 +1,90 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from selenium.webdriver.common.by import By
+
+from tests.ui_tests.app.helpers.locator import Locator, TemplateLocator
+from tests.ui_tests.app.page.common.table.locator import CommonTable
+
+
+class HostListLocators:
+    """Cluster List page elements locators"""
+
+    class Tooltip:
+        apps_btn = Locator(By.XPATH, "//mat-icon[text()='apps']/ancestor::a", "Apps button")
+        host_add_btn = Locator(By.XPATH, "//button[@adcm_test='create-btn']", "Host add button")
+
+    class CreateHostPopup:
+        block = Locator(By.XPATH, "//mat-dialog-container", "Popup block")
+
+        # PROVIDER
```

Comment:
Host creation popup has parts that are about the provider (create new or choose existing one), host (fqdn + save/cancel buttons), and cluster (choose cluster)
It may be subclasses, but then path to it will be very long
I'll consider using subclasses as far as it will be a separate class